src/interpreter/vm/DurianObject.cpp

```diff
@@ -22,6 +22,12 @@ DurianObject::DurianObject(bool bval) :
     value.bval = bval;
 }
 
+DurianObject::DurianObject(int32_t len, unsigned char *sval) :
```

Comment:
I would prefer to have the `:` for member init on a new line, e.g.
```cpp
DurianObject::DurianObject(int32_t len, unsigned char *sval)
    : type(DurianType::String)
    , otherMember(initVal)
```

At least to me it makes it a little clearer that this is in fact member initialization and also lets you line up the `:` on the first line with the `,` on later lines, which looks nice and symmetric.